python/py-pythonz/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        berdario pythonz 1.11.4 pythonz-
+name                py-${github.project}
+
+github.tarball_from tags
+
+platforms           darwin
+license             MIT
```

Comment:
if a port doesn't install architecture-dependent stuff, please add ```supported_archs     noarch```. This appears to be the case for ```py-pythonz```, I haven't checked the others.

Reply:
How do I check this?

Reply:
you can do ```sudo port -dvt <portname> &> log``` and then check the file. If nothing gets compiled and it's only Python code that is installed, which is often the case, then you should add ```supported_archs noarch```.